lib/ContainerManager.js

```diff
@@ -392,15 +408,26 @@ Server.prototype.routes.get['containers'] = function(req, res, next) {
       infos = infos.map(function(info){
         return {
           id: info.Id,
-          name: info.Name,
+          name: info.Name.substr(1), // get rid of initial '/' character
           state: info.State,
           ports: info.NetworkSettings.Ports
         }
       })
 
+      function stateReducer(running){
+        return function(count, info){
+          var state = info.state || info.State
```

Comment:
It's not docker - this code allowed me to test/run with both the direct `infos` objects before being mapped in the above lines, or on my mapped object with `{id, name, state, ports}`. So in the case, `State` becomes `state`